ministryofjustice/correspondence_tool_staff
# CT-3039 Fix the bugs and wording change based on QA result

# Description

The PR is to fix the bugs being found during the QA
- Wording changes for the case transition  record when a linked case has been removed 
- Fixed the bug when deleting ICO appeal case
- Added the parts of cleaning up the linked_cases record when an dependent case has been removed e.g. 
deleting a ICO appeal should clear up all those related links in the linked_cases table
 
## Self-review checklist


### Screenshots


### Related JIRA tickets

https://dsdmoj.atlassian.net/secure/RapidBoard.jspa?rapidView=25&modal=detail&selectedIssue=CT-3039
### Deployment


### Manual testing instructions

Login as London user
Delete a FOI case 
2 - Link a FOI case1  to another FOI case2
3 - you cannot delete case1 without removing the link to case2 
4 - if you remove the link to case2,  then delete case1 afterwards,  the detail of case2 should still be loaded with notes for indicating the removal of case1

Delete a FOI case which has an ICO appeal case against with
1- you cannot delete a FOI case without deleting the ICO appeal case
